environment/learnrails/calculator.rb

```diff
@@ -0,0 +1,30 @@
+class Calculator
```

Comment:
please move definition of  class Calculator in specific file e.g.  environment/learnrails/classes/calculator.rb and in other places in code base you can use it by usage of require
require 'environment/learnrails/classes/calculator'